src/components/card.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 
-import {getTopRatedFilms} from '../services/api.js';
+// import {getTopRatedFilms} from '../services/api.js';
 
 // console.log(getTopRatedFilms);
 
-getTopRatedFilms().then(result => {
+/* getTopRatedFilms().then(result => {
```

Comment:
@Mmarvinn never put commented code in a PR

Reply:
fix